packages/react/src/component-tests/IcTooltip/IcTooltip.cy.tsx

```diff
@@ -240,6 +241,20 @@ describe("IcTooltip visual regression and a11y tests", () => {
     });
   });
 
+  it("should render a IcTooltip with a large label", () => {
+    mount(<LargeLabel />);
+
+    cy.checkHydrated(TOOLTIP_SELECTOR);
+    cy.get("button").realHover("mouse");
+
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "very-large-label",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.004),
```

Comment:
```suggestion
      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.081),
```
ran locally & threshold needs to be higher i think

Reply:
implemented suggested threshold